tests/queries/0_stateless/01824_prefer_global_in_and_join.sql

```diff
@@ -49,7 +49,7 @@ join (
     where d1.id  > 0
     order by d1.id
 ) s0 using id;
-
+set reorder_joins=0;
```

Comment:
What happens when the setting in enabled? Does a query in the tests break with new default setting value?